data/src/main/java/com/linkedin/data/codec/AbstractJacksonDataCodec.java

```diff
@@ -213,12 +224,35 @@ public DataList readList(InputStream in) throws IOException
     return parse(_factory.createParser(in), mesg, locationMap);
   }
 
-  public void objectToJsonGenerator(Object object, JsonGenerator generator) throws IOException
+  /**
+   * Convert an object to Json format representation.
+   * @param object the object that needs to be converted
+   * @param generator the generator that could generate Json output based on the object
+   * @param mapKeyOrdered if the object contains map element (which could be nested map), can set this flag to order the
```

Comment:
If true, then map elements in the object (can be the object itself or its nested elements) will be sorted by Key in the generated Json.